taipy/core/scenario/_scenario_manager.py

```diff
@@ -416,7 +416,7 @@ def _get_children_entity_ids(cls, scenario: Scenario) -> _EntityIds:
         submissions = _SubmissionManagerFactory._build_manager()._get_all()
         submitted_entity_ids = list(entity_ids.scenario_ids.union(entity_ids.sequence_ids, entity_ids.task_ids))
         for submission in submissions:
-            if submission.entity_id in submitted_entity_ids:
+            if submission.entity_id in submitted_entity_ids or submission.entity_id == scenario.id:
```

Comment:
not sure if I understand this part :/

Reply:
Previously, when we find children submissions of a scenario, we only check if the `submission.entity_id` is one of the children tasks or sequences.

However, the `submission.entity_id` can also be the scenario itself.